config/locales/it.yml

```diff
@@ -1,51 +1,100 @@
 it:
+  activemodel:
+    attributes:
+      spree/order_cancellations:
+        quantity: Quantà
+        state: Status
+        shipment: Trasporto
```

Comment:
I think shipment is usually "Spedizione".
We ned to keep coherence between translations